.env.example

```diff
@@ -66,7 +66,7 @@ SAML_CLIENT_SECRET_VERIFIER=
 PGSSLMODE=
 
 # Define which hostnames are expected for the app to work on
-ALLOWED_HOSTNAMES='"cal.com","cal.dev","cal-staging.com","cal.community","cal.local:3000","localhost:3000"'
+ALLOWED_HOSTNAMES='"cal.com","cal.dev","cal-staging.com","cal.community","cal.local:3000","localhost:3000","com","dev","community","local:3000"'
```

Comment:
Why this changes?

Reply:
this is required to identify the OrgDomain or OrgSlug correctly when `acme.cal.com` is redirected to `acme.com` , acme is recognized as OrgDomain in `acme.com` url

Reply:
We shouldn't change this. AFAIU, we don't want to support acme.com. 

acme.com will be the organization's own website and it has nothing to do with us.

Reply:
if we don't append "com" to ALLOWED_HOSTNAMES,
the validation here fails https://github.com/calcom/cal.com/blob/ad62f9d2bdb56a0374a30c45bd939546de25e8e0/packages/features/ee/organizations/lib/orgDomains.ts#L35
so getOrgSlug() fn will return null, instead of slug

Reply:
handled this in alternate method , without appending env variable